### PR TITLE
Add alpine stack and builder

### PR DESCRIPTION
**Pull request by jromero**
Signed-off-by: Javier Romero <jromero@pivotal.io>

### CODE REVIEW

File: apps/java-maven/mvnw [hunk 1]
```diff
@@ -8,7 +8,7 @@
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
 #
-#    https://www.apache.org/licenses/LICENSE-2.0
+#    http://www.apache.org/licenses/LICENSE-2.0
```
Comment (jkutner) [inline apps/java-maven/mvnw]: ewat?

File: stacks/alpine/base/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.10
+ARG cnb_uid=1000
+ARG cnb_gid=1001
+
+# Install packages that we want to make available at both build and run time
+RUN apk add --update --no-cache bash ca-certificates
```
Comment (jkutner) [inline stacks/alpine/base/Dockerfile]: is this a sin?

File: apps/java-maven/mvnw (hunk 1 above)
Comment (jromero) [reply apps/java-maven/mvnw]: /shrug, I just updated to the latest version of the maven wrapper.

Ref: https://github.com/takari/maven-wrapper/blob/master/mvnw

**Comment by jkutner** [comment]
Maybe outside the scope of this change, but why is the namespace `cnbs/`? is that "CNB samples"? If so, I wonder if we should use `cnb-samples/`.

File: stacks/alpine/base/Dockerfile (hunk 2 above)
Comment (jromero) [reply stacks/alpine/base/Dockerfile]: Could you elaborate?

Comment (jkutner) [reply stacks/alpine/base/Dockerfile]: it joke. just installing bash on alpine

**Comment by jromero** [comment]
> Maybe outside the scope of this change, but why is the namespace `cnbs/`? is that "CNB samples"? If so, I wonder if we should use `cnb-samples/`.

I'm open to discussing this further. I do recall that a `-` is not valid as an org or username in docker hub so it would be `cnbsamples` if that's what we propose. `cnbs` was a namespace we already had and used prior so we chose to stick to that for ease.

File: stacks/alpine/base/Dockerfile (hunk 2 above)
Comment (jromero) [reply stacks/alpine/base/Dockerfile]: ahh, 😆

**Pull request merged**